summingbird-online/src/main/scala/com/twitter/summingbird/planner/OnlinePlan.scala

```diff
@@ -17,8 +17,11 @@
 package com.twitter.summingbird.planner
 
 import com.twitter.summingbird._
+import scala.reflect.ClassTag
+import com.twitter.summingbird.online.option._
+import com.twitter.summingbird.online._
```

Comment:
If we're only getting OnlineDefaultConstants, maybe just

import com.twitter.summingbird.online.OnlineDefaultConstants._